### PR TITLE
fix required follows optional parameter deprecation usermapper

### PR DESCRIPTION
**Pull request by mussbach**
This PR fixes the php 8 deprecation and places optional parameters behind mandatory ones

### How to review/test
run phpstan

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [ ] Tests updated/created
- [ ] Update documentation
- [ ] Link all relevant tickets
- [ ] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php [hunk 1]
```diff
@@ -631,14 +630,14 @@ protected function updateOrgaMode2(Orga $orga, $isToeb = false)
      * Create department.
      *
      * @param string $orgaGwId
```
Comment (breierDemos) [inline demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php:632]:
```suggestion
```

File: demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php [hunk 2]
```diff
@@ -631,14 +630,14 @@ protected function updateOrgaMode2(Orga $orga, $isToeb = false)
...
-     * @param bool   $isToeb
      * @param string $orgaId
+     * @param bool   $isToeb
      *
      * @return Department
      *
      * @throws Exception
      */
-    protected function createDepartmentMode2($orgaGwId, $isToeb = false, $orgaId)
+    protected function createDepartmentMode2($orgaGwId, $orgaId, $isToeb = false)
```
Comment (breierDemos) [inline demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php:640]:
```suggestion
    protected function createDepartmentMode2(string $orgaGwId, $orgaId, $isToeb = false)
```

**Review by breierDemos — CHANGES_REQUESTED** [review]
Some suggestions

File: demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php (hunk 1 above)
Comment (mussbach) [reply demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php:632]: What is your suggestion here @breierDemos ? I do not see any difference

File: demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php (hunk 2 above)
Comment (mussbach) [reply demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php]: done 43549e44

File: demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php (hunk 1 above)
Comment (mussbach) [reply demosplan/DemosPlanUserBundle/Logic/UserMapperDataportGatewayHH.php]: got it, the line was deleted. Done in https://github.com/demos-europe/demosplan-core/commit/43549e44a9dc4e761329fd2132b4fe6c9a151004

**Review by breierDemos — APPROVED** [review]

**Pull request merged**